content/finalist/Code Blooded/index.md

```diff
@@ -34,7 +34,7 @@ _**Application-Type**_ : WEBSITE
 
 _**Tech-Stack**_  :   HTML5, CSS 3, JAVASCRIPT, JQUERY, SQL, DJANGO, BOOTSTRAP, FONT AWESOME. 
 
-_**GitHub-Link**_ : https://github.com/KaranBengani/CRAFTIFY   
+_**GitHub-Link**_ : https://github.com/KaranBengani/KRAFTIFY   
```

Comment:
this might be causing conflict.

Reply:
ya but during the register process i had given an update request which changed the Craftify to Kraftify